scripts/update_all_products.pl

```diff
@@ -1347,7 +1349,18 @@
 			assign_ciqual_codes($product_ref);
 		}
 
+		my $any_change = $product_values_changed;
 		if (not $pretend) {
+			if (!$any_change) {
+				# Deep compare with original (if we don't already know that a change has been made)
+				$any_change = !Compare($product_ref, $original_product);
+			}
+			if (!$any_change) {
+				print STDERR ". Skipped";
+			}
+		}
+
+		if ($any_change) {
```

Comment:
is'nt it:
```suggestion
		if ($any_change and (!$pretend)) {
```

Reply:
@alexgarel it's @john-gom who did this change, I'm not familiar with this part of the codebase